root/frontend/src/components/CalendarHeader/CalendarHeader.tsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import { format } from 'date-fns';
+
+import useStyles from './CalendarHeader.style';
+
+import Button from '../Button/Button';
+import { LEFT_CONTROL_SYMBOL, RIGHT_CONTROL_SYMBOL } from './constants';
+
+type PropTypes = {
+  currentSelectedMonth: Date,
+  prevMonthControl: () => void,
+  nextMonthControl: () => void,
+};
+
+type LabelTypes = {
+  month: string,
+  year: string,
+};
+
+function CalendarLabel({ month, year }: LabelTypes) {
+  return (
+    <span data-testid="calendar-label">
+      <b>{month}</b>
+      {` | ${year}`}
+    </span>
+  );
+}
+
```

Comment:
Would it be better to move `CalendarLabel` into a separate folder for better organization?

Reply:
Yeah, it probably would. I will refactor it out.